docker-compose.yml

```diff
@@ -0,0 +1,30 @@
+version: "3.3"
+services:
+  aoc:
+    build: .
+    depends_on:
+      - postgres
+    restart: unless-stopped
+    networks:
+      - aoc-net
+    secrets:
+      - settings
+
+  postgres:
+    image: 'postgres:13.1-alpine'
+    env_file:
+      - db.env
+    volumes:
+      - db-data:/var/lib/postgresql/data
+    networks:
+      - aoc-net
+
+networks:
+  aoc-net:
+
+volumes:
+  db-data:
+
```

Comment:
`- db-data`

Reply:
Seems like it isn't allowed but I'm going to go ahead and just remove it since nothing else will be sharing this volume: https://docs.docker.com/compose/compose-file/#volume-configuration-reference

Reply:
Wait really? That's ugly

Reply:
That's where I got it and the network syntax from 🤷